app/views/claims/reset_claim.erb

```diff
@@ -16,7 +16,7 @@
         You can continue to complete an application to check your eligibility and apply for a payment.
       </p>
 
-      <%= form_with(url: claim_path(current_journey_routing_name, "current-school"), method: :get) do %>
+      <%= form_with model: @form, url: reset_claim_path do %>
         <div class="govuk-form-group">
           <%= hidden_field_tag :skip_landing_page, 'true' %>
```

Comment:
Does this need updating to be `f.hidden_field` otherwise it will come through in the params as `skip_landing_page` rather than `claims[:skip_landing_page]` and we've changed `def skip_landing_page?` to look for this nested under `claim`